fiftyone/server/lightning.py

```diff
@@ -423,6 +425,23 @@ async def _do_distinct_pipeline(
     return values
 
 
+def _add_search(query: DistinctQuery):
+    # strip chars after 24
```

Comment:
Does this mean sidebar only filters by first 24 chars? So long file path filters could potentially not include the file name?

What's the purpose of limiting to the first 24

Reply:
Good comment. I refactored this incorrectly. It should only apply to ObjectId fields

I think I will add frontend validation to omit these queries completely. That is a more holistic approach.

An object id is always a 24 char hexadecimal string. The search results can be omitted when the search does not comply

Reply:
I will undraft when I'm happy with the solution